### PR TITLE
Replaced the team card with a button instead

### LINKED ISSUE #53: Replace the create team card with a button
The create team card takes up a lot of space which can be solved by making it a button instead

### PR DESCRIPTION
**Pull request by JackyTu99**
this fixes #53

### CODE REVIEW

File: src/components/CardNewTeam.vue [hunk 1]
```diff
@@ -72,9 +73,17 @@ export default {
...
         })
       }
     }
   }
 }
 </script>
+
+<style>
+.q-pa-md {
+  text-align: center;
```
Comment (dariosalvi78) [inline src/components/CardNewTeam.vue:87]: remove these lines and add [text-center](https://quasar.dev/style/typography) to the classes of where you want the text to be centred

File: src/components/CardNewTeam.vue [hunk 2]
```diff
@@ -29,7 +30,8 @@ export default {
   name: 'CardNewTeam',
   data () {
     return {
-      teamName: ''
+      teamName: '',
+      prompt: false
```
Comment (dariosalvi78) [inline src/components/CardNewTeam.vue:34]: let's rename this to `promptTeamName`

**Review by dariosalvi78 — CHANGES_REQUESTED** [review]

File: src/components/CardNewTeam.vue (hunk 1 above)
Comment (JackyTu99) [reply src/components/CardNewTeam.vue:87]: the CSS is to center the button on the page

File: src/components/CardNewTeam.vue (hunk 2 above)
Comment (dariosalvi78) [reply src/components/CardNewTeam.vue:34]: change it also in the HTML part!

**Review by dariosalvi78 — APPROVED** [review]

**Pull request merged**